linters/nancy/parse.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+"""
+nancy sleuth --output=json looks like this:
+
+```
+{
+  ...,
+  "vulnerable": [
+    {
+      "Coordinates": "pkg:golang/golang.org/x/text@v0.3.7",
+      "Reference": "https://ossindex.sonatype.org/component/pkg:golang/golang.org/x/text@v0.3.7?utm_source=nancy-client&utm_medium=integration&utm_content=1.0.41",
+      "Vulnerabilities": [
+        {
+          "ID": "CVE-2022-32149",
+          "Title": "[CVE-2022-32149] CWE-400: Uncontrolled Resource Consumption ('Resource Exhaustion')",
+          "Description": "An attacker may cause a denial of service by crafting an Accept-Language header which ParseAcceptLanguage will take significant time to parse.",
+          "CvssScore": "7.5",
+          "CvssVector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
+          "Cve": "CVE-2022-32149",
+          "Reference": "https://ossindex.sonatype.org/vulnerability/CVE-2022-32149?component-type=golang&component-name=golang.org%2Fx%2Ftext&utm_source=nancy-client&utm_medium=integration&utm_content=1.0.41",
+          "Excluded": false
+        }
+      ],
+      "InvalidSemVer": false
+    }
+  ]
+}
+```
+"""
+
+import json
+import sys
+
+
+def to_result_sarif(
```

Comment:
general question - do we have/plan to have any kind of type safety here? I imagine we could define a sarif type and then annotate this to match it quite easily.

Reply:
we should, but i'm not familiar with mypy or how to set up pytype